cluster/addons/dns/kubedns-sa.yaml

```diff
@@ -2,6 +2,7 @@ apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: kube-dns
+  namespace: kube-system
```

Comment:
This is already added on upstream (#44178). If this is needed in 1.6, we should cherrypick that PR instead of creating a new one. @bowei